extensions/linz/README.md

```diff
@@ -107,17 +107,18 @@ This expands on the [provider object in the STAC spec](https://github.com/radian
 
 These fields apply to assets within both items and collections.
 
-| Field Name              | Type   | Description                                                                                                                     |
-| ----------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| assets/\*/created       | string | **REQUIRED**. Creation date and time of the asset, in UTC.                                                                      |
-| assets/\*/updated       | string | **REQUIRED**. Date and time the asset was last updated, in UTC.                                                                 |
-| assets/\*/file:checksum | string | **REQUIRED**. See [reference](https://github.com/stac-extensions/file/blob/v2.0.0/README.md#checksums).                         |
-| linz:language           | string | **REQUIRED**. [RFC 5646 language tag](https://datatracker.ietf.org/doc/html/rfc5646), for example `en-NZ` (New Zealand English) |
+| Field Name    | Type   | Description                                                                                                                     |
+| ------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| created       | string | **REQUIRED**. Creation date and time of the asset, in UTC.                                                                      |
+| updated       | string | **REQUIRED**. Date and time the asset was last updated, in UTC.                                                                 |
+| file:checksum | string | **REQUIRED**. See [reference](https://github.com/stac-extensions/file/blob/v2.0.0/README.md#checksums).                         |
+| linz:language | string | **REQUIRED**. [RFC 5646 language tag](https://datatracker.ietf.org/doc/html/rfc5646), for example `en-NZ` (New Zealand English) |
 
 ## Extensions
 
 This extension includes these other extensions:
 
+- [processing](https://github.com/stac-extensions/processing)
```

Comment:
This is already part of another PR, so it probably shouldn't be here, right?

Reply:
Converted to draft until the other PR merges. Will keep them separate next time.